index.html

```diff
@@ -190,8 +190,26 @@ <h3><i class="im im-trophy"></i>ゴールドスポンサー</h3>
           </a>
         </section>
 
+        <section class="gold-sponsor">
+          <a href="https://www.optimind.tech/">
+            <img
+              src="/assets/logos/optimind.png"
+              alt="株式会社オプティマインド"
+              height="100"
```

Comment:
[解像度は長辺400pxまで](https://ngk2020s.netlify.com/call-for-sponsors.html) です。
このロゴは横長なので横を400pxまでにするか、横が400pxに収まるように画像自体をリサイズしてください。
```suggestion
              width="400"
```